crates/sui-indexer/src/indexer_reader.rs

```diff
@@ -392,6 +392,28 @@ impl IndexerReader {
         Ok(system_state)
     }
 
+    /// Retrieve the system state data for the given epoch. If no epoch is given,
+    /// it will retrieve the last known epoch's data and return the system state.
```

Comment:
I'm a little confused by what "last known epoch" means here -- would the current epoch be considered the "last known epoch" and in that case, would we expect its `system_state` field to contain anything? I was under the impression that it would be empty, meaning this function would just fail when passed `None` for the `epoch`.

Reply:
In any case, if this is for GraphQL, we probably don't need this as the reader APIs for GraphQL are handled elsewhere.

Reply:
it's suggested comments https://github.com/MystenLabs/sui/pull/15434#discussion_r1432001686
is "last know" -> latest slightly better?

per the question, it's discussed under this thread 
https://mysten-labs.slack.com/archives/C05KSEPP27M/p1703034203175009?thread_ts=1702942191.241669&cid=C05KSEPP27M